site/using-weave/host-network-integration.md

```diff
@@ -38,6 +38,20 @@ Exposed addresses can also be added to weaveDNS by supplying fully qualified dom
     host2$ weave expose -h exposed.weave.local
     10.2.1.132
 
+###<a name="routing"></a>Routing from another host
+
+Having run `weave expose` as above, you can use Linux routing to give
+access to the Weave Network from hosts that are not running Weave Net:
+
```

Comment:
After running `weave expose`, you can use Linux routing to provide access to the Weave network from hosts that are not running Weave Net: